integration-libs/opf/payment/root/components/opf-payment-verification/opf-payment-verification.component.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, OnDestroy, OnInit, ViewContainerRef } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { HttpErrorModel } from '@spartacus/core';
+
+import { Observable, Subscription } from 'rxjs';
+import { concatMap } from 'rxjs/operators';
+
+import { GlobalFunctionsDomain } from '@spartacus/opf/global-functions/root';
+import { KeyValuePair, OpfPage } from '../../model';
+import { OpfPaymentVerificationService } from './opf-payment-verification.service';
+
+@Component({
+  selector: 'cx-opf-verify-payment',
+  templateUrl: './opf-payment-verification.component.html',
+})
+export class OpfPaymentVerificationComponent implements OnInit, OnDestroy {
```

Comment:
[MAJOR]

JS of this component (and its indirect dependencies like `OpfPaymentVerificationService` and `OpfResourceLoaderService`) is now eagerly loaded in main.js on every page. It unnecessarily slows down the load of SEO-critical pages like homepage, PDP, PLP.

Note: it happens only when the OPF feature is installed by the customer.

Instead, I suggest to move out as much a spossible of the logic of this component to a lazy-loadable service/facade. Thanks to that, only the component's definition (and a tiny HTML template) will be loaded in main.js, but the actual logic of this component (and its indirect dependencies) will be lazy-loaded only when entering the payment-verification page.